woocommerce/woocommerce
# Move product-editor CES-related components and utilities

### All Submissions:






### Changes proposed in this Pull Request:



This PR moves the remaining product-editor CES-related components and utilities to `@woocommerce/product-editor`.

**No functionality should be impacted by this change... this is just a package file refactoring.**

Fixes #36722.



### How to test the changes in this Pull Request:




Note: make sure tracking is enabled under WooCommerce > Settings > Advanced > WooCommerce.com

In order to enable the new (form-based) product editor, enable the `new-product-management-experience` feature using the WCA Test Helper.

Make sure the product tour modal and feedback footer show by deleting the `woocommerce_product_tour_modal_hidden` option.

In order for CES snackbars and such to show up, you need to have not already dismissed/completed the CES for a particular feature. To reset things, you can delete the `woocommerce_ces_shown_for_actions` transient (use the WCA Test Helper).

1. Load this branch and run `pnpm install && pnpm run build` and make sure everything builds correctly.
2. Verify the CES footer on new (form-based) product editor page works. The "Share feedback" button should bring up a feedback modal.

<img width="1116" alt="Screenshot 2023-03-24 at 16 55 07" src="https://user-images.githubusercontent.com/2098816/227638018-a2078606-22c3-4b05-b6db-447f0ee57e2e.png">


3. Open your console and check if the following exist under `window.wc.productEditor` (there will be more, these are the new ones added in this PR):

- `NEW_PRODUCT_MANAGEMENT_ENABLED_OPTION_NAME `
- `__experimentalProductMVPCESFooter `
- `__experimentalProductMVPFeedbackModal `
- `__experimentalProductMVPFeedbackModalContainer `
- `__experimentalUseProductMVPCESFooter `



### Other information:




### FOR PR REVIEWER ONLY:
